### PR TITLE
skipper: add support for multiple per-hostname OAuth grant credentials

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
For each (hostname, application) pair configured by a new config item `skipper_oauth2_ui_login_hostnames` it creates a PlatformCredentialsSet for the corresponding application.

A new `secret-combiner` CronJob combines all per-hostname OAuth credentials secrets to a single secret mounted into Skipper pods.

A new config item `skipper_oauth2_ui_login_tokeninfo_keys` configures tokeninfo keys available to the `oauthGrant()` Skipper filter users.

copilot:all

### CODE REVIEW

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -258,6 +258,14 @@ skipper_ingress_encryption_key: ""
 skipper_oauth2_ui_login: "false"
 {{end}}
 
+# Comma-separated list of <hostname>=<application> pairs.
+# E.g. to configure "foo.example.org" and "foo.example.com" hostnames for application "foo" and
+# "bar.example.net" for "bar" set
+# skipper_oauth2_ui_login_hostnames: "foo.example.org=foo,foo.example.com=foo,bar.example.net=bar"
+skipper_oauth2_ui_login_hostnames: ""
```
Comment (AlexanderYastrebov) [inline cluster/config-defaults.yaml:265]: `skipper_oauth2_grant_hostnames`?

File: cluster/config-defaults.yaml [hunk 2]
```diff
@@ -258,6 +258,14 @@ skipper_ingress_encryption_key: ""
...
+# Comma-separated list of tokeninfo keys to retain
+skipper_oauth2_ui_login_tokeninfo_keys: ""
```
Comment (AlexanderYastrebov) [inline cluster/config-defaults.yaml:267]: `skipper_oauth2_grant_tokeninfo_keys`?

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**